samples/esb/esb_monitor/README.rst

```diff
@@ -40,6 +40,60 @@ All LEDs:
 
    To disable LEDs, unset the :kconfig:option:`CONFIG_LED_ENABLE` Kconfig option.
```

Comment:
Is this a sample-specific Kconfig option? At least, it does not link and is not found in the Kconfig search.